sync_csv.sh

```diff
@@ -79,15 +79,15 @@ fi
 bq show httparchive:${ptable} &> /dev/null
 if [ $? -ne 0 ]; then
   echo -e "Submitting new pages import ${ptable} to BigQuery"
-  bq --nosync load $ptable gs://httparchive/${archive}/pages.csv.gz $BASE/schema/pages.json
```

Comment:
This will make the pages submit synchronous and block request upload, is that what you meant to do? I assume you can make it --replace alongside --nosync.

Reply:
Yeah, the pages table is relatively small and loads quickly. Also by loading synchronously, we're able to see any errors.

Reply:
OK, sgtm, just wanted to make sure this is intentional.